scripts/Muon/GUI/Common/contexts/muon_data_context.py

```diff
@@ -164,10 +164,9 @@ def get_loaded_data_for_run(self, run):
 
     def loaded_workspace_as_group(self, run):
         if self.is_multi_period():
-            workspace_group = WorkspaceGroup()
-            for workspace_wrapper in self._loaded_data.get_data(run=run, instrument=self.instrument)['workspace']['OutputWorkspace']:
-                workspace_group.addWorkspace(workspace_wrapper.workspace)
-            return workspace_group
+            workspace_list = [wrapper._workspace_name for wrapper in self._loaded_data.get_data(
+                run=run, instrument=self.instrument)['workspace']['OutputWorkspace']]
+            return GroupWorkspaces(InputWorkspaces=workspace_list, OutputWorkspace='__temp_group')
```

Comment:
doesn't this use a tmp workspace group name? I thought that breaks project recovery